packages/pricing/src/loaders/connection.ts

```diff
@@ -0,0 +1,27 @@
+import { InternalModuleDeclaration, LoaderOptions } from "@medusajs/modules-sdk"
```

Comment:
TODO: Might be worth extracting a common util out of common loaders and make it accept generics.